### PR TITLE
Dec 214

### PR DESCRIPTION
**Pull request by jonhartzler**
Long content was scrolling in the showcase admin and was affecting the side scrolling. 

scroll -> hidden

### CODE REVIEW

**Comment by danthewolfe** [comment]
:+1:

File: config/style_guides/.javascript-style.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (rfox2) [inline config/style_guides/.javascript-style.json]: What is this file? Just curious.

**Comment by rfox2** [comment]
:+1:

File: config/style_guides/.javascript-style.json (hunk 1 above)
Comment (danthewolfe) [inline config/style_guides/.javascript-style.json]: This tells Hound to not to complain about certain things.

**Pull request merged**